recipes/r-ssh/meta.yaml

```
@@ -0,0 +1,88 @@
+{% set version = '0.7.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-ssh
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/ssh_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/ssh/ssh_{{ version }}.tar.gz
+  sha256: 2b6dc51c6ff8b0c8818f25cf31ca0db1456ec4cd88b90c66147148e337d36f69
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-askpass
+    - r-credentials
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-askpass
+    - r-credentials
+
+test:
+  commands:
+    - $R -e "library('ssh')"           # [not win]
+    - "\"%R%\" -e \"library('ssh')\""  # [win]
+
+about:
+  home: https://docs.ropensci.org/ssh (website), https://github.com/ropensci/ssh (devel)
+  license: MIT
+  summary: Connect to a remote server over SSH to transfer files via SCP, setup a secure tunnel,
+    or run a command or script on the host while streaming stdout and stderr directly
+    to the client.
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - ropensci/ssh
+    - lc5415/ssh
```

Only usernames and conda-forge groups work here.
```suggestion
    - lc5415
```